lib/cocoapods/command/spec/lint.rb

```diff
@@ -56,7 +56,7 @@ def initialize(argv)
           @swift_version   = argv.option('swift-version', nil)
           @skip_import_validation = argv.flag?('skip-import-validation', false)
           @skip_tests      = argv.flag?('skip-tests', false)
-          @test_specs      = argv.option('test-specs', nil).try(:split, ',')
+          @test_specs      = argv.option('test-specs', nil)&.split(',')
```

Comment:
this was using an `activesupport` function that doesn't seem to be available now, but now it isn't needed since we can use `&.` :D